examples/self-diagnostics/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "self-diagnostics"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+publish = false
+
+[dependencies]
+opentelemetry = { path = "../../opentelemetry" }
+opentelemetry_sdk = { path = "../../opentelemetry-sdk", features = ["rt-tokio"]}
+opentelemetry-stdout = { path = "../../opentelemetry-stdout"}
+opentelemetry-appender-tracing = { path = "../../opentelemetry-appender-tracing"}
+tokio = { workspace = true, features = ["full"] }
+serde_json = { workspace = true }
+tracing = { workspace = true, features = ["std"]}
+tracing-core = { workspace = true }
+tracing-subscriber = { version = "0.3.18", features = ["env-filter","registry", "std"]}
+
+[features]
+default = ["otel_unstable"]
+otel_unstable = ["opentelemetry/otel_unstable"]
```

Comment:
what is requiring the unstable feature flag?

Reply:
yeah copy-paste, will remove it.